pom.xml

```diff
@@ -51,7 +51,8 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Build properties -->
-        <cs.jdk.version>11</cs.jdk.version>
+        <cs.jdk.version>14</cs.jdk.version>
```

Comment:
This would cause PR to be built against jdk14, given what got broken and fixed to make it work for jdk11 - moving to a newer jdk may be more work.